hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -0,0 +1,155 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "entropy_src",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  bus_host: "none",
+  interrupt_list: [
+    { name: "es_noise_valid"
+      desc: "Asserted when noise source bits are available."}
+    { name: "es_noise_fifo_err"
+      desc: "Asserted when a FIFO error occurs."}
+  ]
+  regwidth: "32",
+  registers: [
+    { name: "ESCONF",
+      desc: "ENTROPY_SRC configuration register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [
+        { bits: "0",
+          name: "Master enable",
+          desc: "Enable ENTROPY_SRC noise generation function."
+        }
+      ]
+    },
+    { name: "ESREV",
+      desc: "ENTROPY_SRC revision register",
+      swaccess: "ro",
+      hwaccess: "hro",
+      fields: [
+        { bits: "23:16",
+          name: "ENTROPY_SRC Chip Type",
+          desc: "Read of this register shows the type of chip using this ENTROPY_SRC HW.",
+          resval: "0x1"
+        }
+        { bits: "15:8",
+          name: "ENTROPY_SRC HW revision",
+          desc: "Read of this register shows the level of the ENTROPY_SRC HW."
+          resval: "0x2"
+       }
+        { bits: "7:0",
+          name: "ENTROPY_SRC ABI revision",
+          desc: "Read of this register shows the level of the ENTROPY_SRC ABI."
+          resval: "0x1"
+        }
+      ]
+    },
+    { name: "ESMINENT",
+      desc: "ENTROPY_SRC min-entropy estimate register",
+      swaccess: "rw",
+      hwaccess: "hrw",
+      fields: [
+        { bits: "31:0",
+          name: "ENTROPY_SRC Min-entropy",
```

Comment:
This probably needs a bit more explanation in the document. How would SW use this value? How is this value generated? It appears to be a read-only constant (agreeing with @vogelpi above).

Reply:
As already mentioned, considering removing this register.